package.json

```diff
@@ -38,7 +38,7 @@
     "sinon": "^3.3.0"
   },
   "peerDependencies": {
-    "@sentry/node": "^5.23.0"
+    "@sentry/node": ">= 5.24.2 <7"
```

Comment:
I don't think we should have sentry as a peer dependency of clay log at all. Is it possible to fix the issue we're having without it?